Cargo.toml

```diff
@@ -6,7 +6,6 @@ members = [
     "examples/nat64",
     "examples/ping4d",
     "examples/pktdump",
-    "examples/signals",
```

Comment:
@drunkirishcoder It's been awhile, and I remember us discussing why we're removing signal-handling out of the lib. Though, should we show an example (with external dep/lib) where a running application responds to a unix signal?

Reply:
in the old runtime, we provide our signal handling api that's integrated into the tokio. the new runtime is completely signal lib free. the user is free to choose whatever method they prefer to stop the program. whatever they do will be completely independent of capsule and we don't care. our examples use [`signal_hook`](https://docs.rs/signal-hook/latest/signal_hook/) for `ctrl-c` stop.

Reply:
@drunkirishcoder i know. Wasn’t sure if we wanted to have an example to show how you could do it yourself now, but I think you’re saying it’s open-ended enough.